src/Integration.UnitTests/ProjectToLanguageMapperTests.cs

```diff
@@ -144,6 +146,34 @@ public void GetAllBindingLanguagesForProject_NotCMakeProject_UnknownLanguage()
             actualLanguage.Should().BeEquivalentTo(Language.Unknown);
         }
 
+        [TestMethod]
```

Comment:
All new tests for copied from the CMake / JSTS tests and changed to check secrets.